src/corelibs/U2Algorithm/src/misc/FindAlgorithm.cpp

```diff
@@ -155,6 +156,10 @@ static void findInAmino(FindAlgorithmResultsListener* rl,
     QByteArray revPattern(pattern);
     TextUtils::reverse(revPattern.data(), patternLen);
 
+    if (!DynTable::isAcceptableMatrixDimensions(width, height)) {
+        coreLog.error(QObject::tr("Matrix with given dimensions too big for calculation."));
```

Comment:
The text was not fixed.
Do not use QObject::tr when you have a better context: use some of FindAlgorithm QObject classes.

Reply:
FindAlgorithm is not a QObject successor.

Reply:
Ok